src/main/kotlin/org/b12x/gfe/core/controller/PrefsCore.kt

```diff
@@ -0,0 +1,15 @@
+package org.b12x.gfe.core.controller
+
+import org.b12x.gfe.GSG
+import org.b12x.gfe.plugins.gfesearch.controller.locistategfesearch.PrefsGfeSearch
+import java.util.prefs.Preferences
+
+object PrefsCore {
+    var prefs: Preferences = Preferences.userNodeForPackage(GSG::class.java)
+
+    fun nuclearOption() {
+        prefs.removeNode()
```

Comment:
It's because of how preferences work. In the Window's registry it creates a parent node, then chains all other preferences from it. Having gone registry spelunking before, having multiple top level nodes is really annoying. This will put all preferences under the GSG node, which makes sense to me.